products/compute/terraform.yaml

```diff
@@ -1677,8 +1680,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         input: false
         default_from_api: true
         custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
-      logConfig: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Are we able to keep the field O+C? I'm a little hesitant to start catching drift out of the blue, especially on our last `2.X` release. However, it's beta-only so it's allowed imo.